src/button/interfaces.ts

```diff
@@ -146,11 +108,6 @@ export interface ButtonProps extends BaseComponentProps {
    */
   onFollow?: CancelableEventHandler<ButtonProps.FollowDetail>;
```

Comment:
Do we want to inherit `onFollow`? To my understanding the reason to have it in the Button component is for cases where using `href` and therefore clicking the button navigates to a different route, which is not a use case for the Toggle Button

Reply:
updated